### ISSUE TITLE
Henkaku unable to start when Vita card is not inserted (error occurred, C2-12828-1)

### ISSUE THREAD

**Issue by mickael28** [henkaku/henkaku.github.io#21]
As I couldn't use the card with the previous issue. I've now trying to launch henkaku.xyz/go for a long period but I keep on getting:

"An error has occurred in the following application.
(C2-12828-1)
Internet Browser"

I tried deleting cookies, history, restarting, ... but keep on getting it. 
I researched online for a while but I cannot see what could be going on or what I could do? Can Henkaku handle this issue? And do you know what it is or how to workaround it in the time being?

Thank you.

**Comment by yifanlu** [comment]
You need a memory card or internal memory to use henkaku.

**Comment by mickael28** [comment]
Yes, my PS Vita has the 1GB internal memory intact, but even like that Henkaku doesn't load. Shouldn't it, has that internal memory is free?

**Comment by mickael28** [comment]
Ah, sorry, I read yesterday in a few forums that the original Vita had internal memory, but reading more about the internal memory now, and just found out that the original Vita has internal memory but I think Sony configured it to not be accessible, so it seems as if Henkaku won't be able to install in that scenario. 

Bad luck, as this was the workaround I was trying to use due to issue #20. So at present, I've got a card that was used in a PSTV 3.63 system but when I have that inserted and try to load henkaku.xyz/go... the browser just freezes in a white screen :(

If you can think of any workaround, or anything I could try from my end to past through that error, pls let me know and I'll give it a go.

Thank you!

**Comment by yifanlu** [comment]
You need to format the card.

Please use the forums or the chatroom for support. Github is for reporting henkaku bugs.

**Comment by mickael28** [comment]
Yes, but I need to use the card in both consoles (PS Vita 3.60 and PSTV 3.63) which doesn't seem currently supported by Henkaku, hence that I reported it in #20, in case you had the knowledge to see what could be going on and find a fix for that... 

Otherwise, I'll have to buy two different cards, which at present can't do.

**Comment by yifanlu** [comment]
This is not a bug. HENkaku requires either a memory card or internal memory. A memory card that is not usable (locked to another console at > 3.60) is not a valid memory card.

**Comment by yifanlu** [comment]
Think about it this way: with your locked memory card, you can't install any games so why do you think you can install henkaku?

**Comment by mickael28** [comment]
I am trying to find a workaround to the issue as a new app was released recently to allow bypass this issue.

It's this one, https://github.com/cnsldv/CardUnlock

The only thing is that they say we need to install Henkaku without the card installed, which is what I was trying to do on a 1000 model Vita and it failed. 

Checking on Reddit further (https://www.reddit.com/r/vitahacks/comments/695xq3/release_cardunlock_swap_memory_cards_between/), it seems some people suggested that if Henkaku gets updated to support installing to ur0 then CardUnlock would work in that model as well, but at present they said Henkaku doesn't support that (which is kind of what I was trying to explain in the issue I raised #20, but I didn't know the technical terms).

I don't even know if that's technically possible, but if it is, it'd help 1000 Vita users to bypass that limitation when used together with CardUnlock.

Thanks.
